SECURITY.md

```diff
@@ -0,0 +1,82 @@
+# Security Policy
+
+
+## Supported Versions
+Please see [Releases](https://github.com/stacks-network/stacks-blockchain/releases). We recommend using the [most recently released version](https://github.com/stacks-network/stacks-blockchain/releases/latest).
+
+
+## Reporting a vulnerability
+**Please do not file a public issue or PR mentioning the vulnerability.**
+
+We’ve partnered with [ImmuneFi](https://immunefi.com/) to reward honest researchers who find and responsibly disclose security vulnerabilities in our critical code. We offer bounties payable in the Stacks token ([STX](https://www.coingecko.com/en/coins/stacks)) for accepted, high-quality submissions.
```

Comment:
r/We/The Stacks Foundation

^ this substition applies for the entire PR.

Reply:
Language has been adjusted